docs/ReleaseNotes.md

```diff
@@ -3,8 +3,9 @@ Release Notes for Version 14
 
 Build 027
 -------
-Published as version 14.17.1
-
+Published as version 14.18.0
+New Features:
+* Updated pseudo string to be encapsulated.
```

Comment:
Should be more descriptive. Something like, "Updated pseudo strings to be encapsulated with square brackets so that you can tell when the resource string starts and ends."

Reply:
updated it.